contrib/win32/win32compat/win32_pty.c

```diff
@@ -180,10 +145,12 @@ int exec_command_with_pty(int * pid, char* cmd, int in, int out, int err, unsign
 		errno = EOTHER;
 		goto done;
 	}
+
 	*pid = pi.dwProcessId;
 	ret = 0;
 
 done:
+	SetConsoleCtrlHandler(NULL, TRUE);
```

Comment:
@bagajjal , is this okay to call this on shellhost case? or I need to add a condition to call it only for conpty case?

Reply:
I think we do this inside shellhost but you can verify it once..
When you are using ssh-shellhost.. 
1) launch ping -t bing.com
2) Now issue CTRL+C
3) If we are able to stop the ping then we are good..

Reply:
confirm that ssh-shellhost has it and CTRL+C works fine.